src/envoy/auth/jwt.cc

```diff
@@ -208,7 +196,38 @@ class EvpPkeyGetter {
   }
 };
 
-// Class to decode and verify JWT. Setup() must be called before
+std::pair<bool, std::string> GetStringFromJson(const rapidjson::Document &d,
```

Comment:
How about reusing envoy's JsonLoader and JsonObject?
https://github.com/lyft/envoy/blob/master/include/envoy/json/json_object.h

Reply:
you mean, replacing all `rapidjson::Document` in this lib with envoy's JsonObject? Or just for this function?

Reply:
yes replacing rapidjson::Document if possible.

Reply:
done